### PR TITLE
docs: add roadmap

### PR DESCRIPTION
**Pull request by jcstr**
Actually i would like to know if there's something new to add, this is what i see to get started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,11 +80,11 @@ Celo accepts a list of files as well as Glob patterns in both `encryption` and `
 
 ## Road map
 - [x] Use standar algorithm to encrypt files
-- [] Enhance file handling with buffers
-- [] Handle errors 
-- [] Show errors properly
-- [] Unit tests
-- [] Packaging
+- [ ] Enhance file handling with buffers
+- [ ] Handle errors
```
Comment (rrivera) [inline README.md]: Errors are handled, I don't think this point is needed

File: README.md [hunk 2]
```diff
@@ -80,11 +80,11 @@ Celo accepts a list of files as well as Glob patterns in both `encryption` and `
...
 ## Road map
 - [x] Use standar algorithm to encrypt files
-- [] Enhance file handling with buffers
-- [] Handle errors 
-- [] Show errors properly
-- [] Unit tests
-- [] Packaging
+- [ ] Enhance file handling with buffers
+- [ ] Handle errors 
+- [ ] Show errors properly
```
Comment (rrivera) [inline README.md]: I would rename it to:

Improve errors messages

File: README.md [hunk 3]
```diff
@@ -80,11 +80,11 @@ Celo accepts a list of files as well as Glob patterns in both `encryption` and `
...
+- [ ] Unit tests
```
Comment (rrivera) [inline README.md]: I would say that Unit tests should be the next thing to do. What do you think about moving this to the top of the list?

Comment (jcstr) [reply README.md]: Of course, let me update.

**Comment by jcstr** [comment]
With this commit i think i'm done with documentation. Now, are we ready for the 'Unit testing' task?

**Pull request merged**